metis/lib/models/file.rb

```diff
@@ -168,6 +168,7 @@ def to_hash(request: nil, file_path: nil)
       file_path ||= self.file_path
 
       {
+        id: self.id,
```

Comment:
What is the use case for sending database Ids back to the user?  I thought we didn't really want consumers relying on that for anything...

Reply:
Because updated_at is not monotonically increasing (it's possible to have many records share the same updated_at), the id is needed to differentiate and order records in case of a tie on updated_at.

Reply:
I thought that the pagination and ordering in QueryBuilder / Query happens on the Sequel dataset before this method is called, not on a set of File hashes....so this shouldn't be required for that functionality to work? Or do you mean that `id` would be used to differentiate on the consumer side, like an ETL script?

Reply:
The path itself might suffice to provide that ordering and avoid exposing the id. On the other hand maybe it is nice to start to use the id on occasion and avoid unnecessary folder lookups.

Reply:
You are right though I can use the file path instead, I'll remove again.